src/adloader.js

```diff
@@ -20,8 +19,7 @@ const _approvedLoadExternalJSList = [
   'inskin',
   'hadron',
   'medianet',
-  'improvedigital',
-  'aaxBlockmeter'
```

Comment:
why aaxBlockmeter was removed from this file?

Reply:
Interesting. Because I didn't make any edit this file. I just removed admatic and pixad line. I pushed the adloader.js file again.